draft-ietf-quic-recovery.md

```diff
@@ -634,7 +634,7 @@ largest_sent_packet:
 : The packet number of the most recently sent packet.
 
 largest_acked_packet:
-: The largest packet number acknowledged in an ACK frame.
+: The largest packet number acknowledged by any ACK frame.
```

Comment:
```suggestion
: The largest packet number acknowledged in the connection so far.
```

Reply:
Isn't this in the packet number space?

Reply:
I think that you mean "packet number space" rather than connection.